scripts/build_python.sh

```diff
@@ -132,6 +134,9 @@ while (($#)); do
             pregen_dir=$2
             shift
             ;;
+        --jupyter-lab | -j)
+            install_jupyterlab=yes
+            ;;
```

Comment:
is there a shift missing here?

Reply:
The others require a "yes" argument hence the shift. In this case I did not see the need to say `-j yes` or `-j no`. I believe the other arguments are due to historical usage so were never changed.